docs/source/modin/supported/dataframe_supported.rst

```diff
@@ -169,7 +169,7 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``ewm``                     | N                               |                                  |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``expanding``               | N                               |                                  |                                                    |
+| ``expanding``               | P                               |                                  | ``N`` if ``axis = 1``                              |
```

Comment:
you also need to add those to the window.rst to make sure doc gen will update those functions.

Reply:
Good catch, will do